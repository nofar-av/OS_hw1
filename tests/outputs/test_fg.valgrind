==27544== Memcheck, a memory error detector
==27544== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==27544== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==27544== Command: /home/omer/OS_wet1/smash
==27544== Parent PID: 27542
==27544== 
==27544== 
==27544== FILE DESCRIPTORS: 4 open at exit.
==27544== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_fg.valgrind
==27544==    <inherited from parent>
==27544== 
==27544== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_fg.err
==27544==    <inherited from parent>
==27544== 
==27544== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_fg.out
==27544==    <inherited from parent>
==27544== 
==27544== Open file descriptor 0:
==27544==    <inherited from parent>
==27544== 
==27544== 
==27544== HEAP SUMMARY:
==27544==     in use at exit: 0 bytes in 0 blocks
==27544==   total heap usage: 36 allocs, 36 frees, 82,696 bytes allocated
==27544== 
==27544== All heap blocks were freed -- no leaks are possible
==27544== 
==27544== For counts of detected and suppressed errors, rerun with: -v
==27544== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
