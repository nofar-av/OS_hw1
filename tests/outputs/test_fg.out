smash> smash> smash> sleep 1& : 28349
smash> smash> 